modin/core/io/text/json_dispatcher.py

```diff
@@ -57,25 +57,20 @@ def _read(cls, path_or_buf, **kwargs):
         empty_pd_df = pandas.DataFrame(columns=columns)
 
         with OpenFile(path_or_buf, "rb", kwargs.get("compression", "infer")) as f:
-            partition_ids = []
-            index_ids = []
-            dtypes_ids = []
-
             column_widths, num_splits = cls._define_metadata(empty_pd_df, columns)
-
             args = {"fname": path_or_buf, "num_splits": num_splits, **kwargs}
-
             splits = cls.partitioned_file(
                 f,
                 num_partitions=NPartitions.get(),
             )
-            for start, end in splits:
+            partition_ids = [None] * len(splits)
+            index_ids = [None] * len(splits)
+            dtypes_ids = [None] * len(splits)
+            for idx, (start, end) in enumerate(splits):
                 args.update({"start": start, "end": end})
-                partition_id = cls.deploy(cls.parse, num_splits + 3, args)
-                partition_ids.append(partition_id[:-3])
-                index_ids.append(partition_id[-3])
-                dtypes_ids.append(partition_id[-2])
-
+                *partition_ids[idx], index_ids[idx], dtypes_ids[idx], _ = cls.deploy(
```

Comment:
Why the extra `_`? Is that value unused?

Reply:
`PandasJSONParser` returns [also](https://github.com/modin-project/modin/blob/7c02bafa1badade0ba9d1916652ded74bc36a1b9/modin/core/storage_formats/pandas/parsers.py#L625) `partitions_columns` value. This is laid down for the future - to implement `lines=False` case. (According to [the comment](https://github.com/modin-project/modin/blob/7c02bafa1badade0ba9d1916652ded74bc36a1b9/modin/core/io/text/json_dispatcher.py#L79))